app/views/layouts/_nav_bar.html.erb

```diff
@@ -1,22 +1,21 @@
 <%
-  t = @teams.nil? ? nil : @teams.map(&:updated_at).join;
+  t = @teams.nil? ? nil : @teams.map(&:updated_at).join
```

Comment:
I think this would be better in a helper method like `cache_key_for_nav`.

Reply:
I honestly do not understand what this block does, so I don't really want to mess with it 😓 

Is it creating a cache_key?  That helps.  I'll take another look.